src/services/erase.js

```diff
@@ -1,11 +1,11 @@
-const sql = require("./sql.js");
+const sql = require("./sql");
 const revisionService = require("./revisions.js");
-const log = require("./log.js");
-const entityChangesService = require("./entity_changes.js");
-const optionService = require("./options.js");
-const dateUtils = require("./date_utils.js");
+const log = require("./log.ts");
```

Comment:
Why does this require an extension?

Reply:
@j-tirelli , indeed not needed. I've removed it.